vividus-extension-selenium/src/main/java/org/vividus/selenium/WebDriverProvider.java

```diff
@@ -76,13 +77,14 @@ public void end()
             String sessionId = WebDriverUtil.unwrap(webDriver, RemoteWebDriver.class).getSessionId().toString();
             try
             {
+                eventBus.post(new BeforeWebDriverQuitEvent());
```

Comment:
does it make sense to add `sessionId` to `BeforeWebDriverQuitEvent` as well?

Reply:
not sure, currently this used in one place we have only BS client which requires sessionId, whereas two others don't